integration_tests/pages/page.ts

```diff
@@ -104,8 +104,11 @@ export default abstract class Page {
     })
   }
 
-  shouldShowBanner(copy: string): void {
-    cy.get('.govuk-notification-banner').contains(copy)
+  shouldShowBanner(heading: string, body?: string): void {
+    cy.get('.govuk-notification-banner').contains(heading)
+    if (body) {
+      cy.get('.govuk-notification-banner').contains(body)
```

Comment:
This will only be able to handle text for the comparison and so, may not work if the html sent to the body is broken up into multiple elements.
There is a routine in `page.ts` - `parseHtml` that I think would allow full HTML comparisons.

Reply:
Very good call -- I've updated `shouldShowBanner` to make use of this. Given a couple of previous tests only cared if the banner _contained_ the text, I've also added an `options` argument: if setting `options.exact` to `false`, the assertion is that the banner _contains_ the text; the default (`exact: true`) asserts that the banner content and text are an exact match.